CHANGELOG.rst

```diff
@@ -46,6 +46,8 @@ Changelog
   :meth:`~cryptography.x509.CertificateSigningRequest.get_attribute_for_oid`.
 * Added support for encoding attributes in certificate signing requests via
   :meth:`~cryptography.x509.CertificateSigningRequestBuilder.add_attribute`.
+* The OS random engine for OpenSSL is now disabled on OpenSSL 1.1.1d and
+  higher because these versions now properly reseed the RNG on fork.
```

Comment:
I'd phrase this as: we now use OpenSSL's builtin CSPRNG instead of our own on OpenSSL 1.1.1d+ because it's finally good.

Reply:
"It's finally good" isn't really what I want to put in a changelog. I think the text describes exactly why and I'm okay with not noting that we literally fought to have this be properly fixed before 1.1.1 was released and lost the fight. 😆

Reply:
Doesn't need to use the "finally good" language, but I think framing it as "we use the builtin thing now" instead of "we disable our thing" is much clearer.